app/ui-react/syndesis/src/app/UI.tsx

```diff
@@ -99,88 +99,118 @@ export const UI: React.FunctionComponent<IAppUIProps> = ({ routes }) => {
                 notificationTimerDelay={8000}
                 removeNotificationAction={onRemoveNotification}
               />
-              <AppLayout
-                onShowAboutModal={toggleAboutModal}
-                appNav={
-                  <AppTopMenu username={'developer'}>
-                    <PfDropdownItem>
-                      <Link
-                        to={'/logout'}
-                        className="pf-c-dropdown__menu-item"
-                        children={t('Logout')}
-                      />
-                    </PfDropdownItem>
-                  </AppTopMenu>
-                }
-                verticalNav={routes.map((route, index) =>
-                  !(route as IAppRouteWithChildrens).childrens ? (
-                    <PfVerticalNavItem
-                      exact={(route as IAppRoute).exact}
-                      label={t((route as IAppRoute).label)}
-                      to={(route as IAppRoute).to}
-                      key={index}
-                      data-testid={`navbar-link-${(route as IAppRoute).to}`}
-                    />
-                  ) : (
-                    <PfVerticalNavItem
-                      label={t(route.label)}
-                      key={index}
-                      to={'#'}
-                    >
-                      {(route as IAppRouteWithChildrens).childrens.map(
-                        (subRoute, subIndex) => (
+              <WithRouter>
+                {({ history, match }) => {
+                  return (
+                    <AppLayout
+                      onShowAboutModal={toggleAboutModal}
+                      onSelectSupport={() => {
+                        history.push('support');
```

Comment:
Erh... sorry I didn't spot this earlier. You should use a resolver to retrieve the right URL, not a hardcoded string. Something like `resolvers.support.root()`